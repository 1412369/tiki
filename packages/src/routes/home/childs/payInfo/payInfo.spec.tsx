import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PayInfo from './payInfo';
describe('snapshot PayInfo', function() {
  it('Should render PayInfo without crashing', function() {
    const wrapper = renderer.create(
      <PayInfo
        location={{
          name: 'CGV Su Van Hanh',
          start: '9:00',
          end: '10:00',
        }}
        calculatePrice="120,000 Ä‘"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
