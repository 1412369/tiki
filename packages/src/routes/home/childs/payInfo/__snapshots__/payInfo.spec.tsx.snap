// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot PayInfo Should render PayInfo without crashing 1`] = `
<div
  className="sc-bdVaJa jWCWAf"
>
  <div
    className="movie-time"
  >
    <span>
      CGV Su Van Hanh
    </span>
    <span>
      9:00
       | 
      10:00
    </span>
  </div>
  <div
    className="movie-total"
  >
    <span>
      Tổng cộng
    </span>
    <span>
      120,000 đ
    </span>
  </div>
</div>
`;
